integ-test/src/test/resources/text_keyword_index.json

```diff
@@ -0,0 +1,16 @@
+{"index": {}}
+{"typeKeyword": "key00", "typeText": "text00", "typeKeywordFieldNoFieldData": "keyword00","typeTextFieldData": "keyFD00", "typeKeywordFieldData":  "textFD00", "textDataFieldNoFields": "textFDNF00","int0": 0}
+{"index": {}}
+{"typeKeyword": "key01", "typeText": "text01", "typeKeywordFieldNoFieldData": "keyword01", "typeTextFieldData": "keyFD01", "typeTextFieldData":  "textFD01OverTen", "textDataFieldNoFields": "textFDNF01", "int0": 1}
+{"index": {},typeKeywordFieldData
```

Comment:
A typo? How it works?

Reply:
Resolved. Issue created https://github.com/opensearch-project/sql/issues/1110